releases/1.0.0/index.md

```diff
@@ -33,12 +33,14 @@ custom_css: releases
 * Added OWL 2 DL review with SHACL-SPARQL for UCO ([*Github Pull Request 102*](https://github.com/casework/CASE/pull/102))
   * (While this is not necessarily a backwards-incompatible change within the scope of UCO, it does identify what end users might not have realized were broader OWL errors.)
 * Removed CASE's IRI base ([*GitHub Pull Request 103*](https://github.com/casework/CASE/pull/103)) 
+* Removed requirement of repeating all property constraints from parent to child classes ([*UCO GitHub Issue 417*](https://github.com/ucoProject/UCO/issues/417))
```

Comment:
This should be on UCO's 1.0.0 release page instead of CASE's.

Reply:
...actually, how about in addition to.  Go ahead and keep it here, but UCO's page needs this noted.